extensions/wikia/CommunityPage/styles/components/_moderators-module.scss

```diff
@@ -0,0 +1,58 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/clearfix';
+@import 'skins/shared/mixins/flexbox';
+@import 'variables';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/base/typography';
+
+// Styles for the moderators module in-page
+.moderators-module {
+	.avatar-container {
```

Comment:
Yes, and because of the differences between all of the modules I leaned towards having them completely separate.